PurchaseOrderNumber (#388)
MewsSystems/gitbook-connector-api: operations/bills.md

| `TaxedUtc` | string | optional | Taxation date of the bill in UTC timezone in ISO 8601 format. |
| `PaidUtc` | string | optional | Date when the bill was paid in UTC timezone in ISO 8601 format. |
| `DueUtc` | string | optional | Bill due date and time in UTC timezone in ISO 8601 format. |

> | `PurchaseOrderNumber` | string | optional | Number of purchase order (document) a buyer sends to a seller after ordering goods or services. |

Reviewer: ```suggestion
| `PurchaseOrderNumber` | string | optional | Unique number of the purchase order from the buyer. |
```

| `DueDate` | [String update value](#string-update-value) | optional | Deadline when [Bill](#bill) is due to be paid. Can be used only with [Bill type](#bill-type) `Invoice`. |
| `VariableSymbol` | [String update value](#string-update-value) | optional | Optional unique identifier of requested payment. Can be used only with [Bill type](#bill-type) `Invoice`. |
| `TaxIdentifier` | [String update value](#string-update-value) | optional | Tax identifier of account to be put on a bill. |

> | `PurchaseOrderNumber` | [String update value](#string-update-value) | optional | Number of purchase order (document) a buyer sends to a seller after ordering goods or services. |

Reviewer: ```suggestion
| `PurchaseOrderNumber` | [String update value](#string-update-value) | optional | Unique number of the purchase order from the buyer. |
```
